content/1_General/Adding_Solution.mdx

```diff
@@ -88,24 +88,33 @@ See [Working With MDX](/general/working-mdx) for additional information.
     ```
 
     Keep file names and solution IDs consistent; namely, USACO IDs should
-    correspond to the URL on
+    correspond to the number at the very end of the URL on
     [usaco.org](http://www.usaco.org/index.php?page=viewproblem2&cpid=690) and
-    file names should be the same as the solution's ID.
+    file names should be the same as the solution's ID. See
+    [Working With MDX](/general/working-mdx#problem-lists) for more examples of
+    IDs.
 
 3.  Add your implementation, following these conventions and the conventions
     [below](#code-conventions):
 
-    - Use _tabs_ to indent.
-    - Lines should not exceed 80 characters.
+    - Use _tabs_ to indent
+    <Info title="Pro Tip">
+	If your code is indented with spaces, use the [Guide Editor](/editor), copy-paste your code in, right-click, click "Command Palette" from the menu that pops up and then search for "Convert Indentation to Tabs". Once you see it, click on it and then copy that new code with the correct indentation into your Github pull request.
+    </Info>
+    
+    - Lines should not exceed 80 characters. If you need to, you can write comments across multiple lines to follow this rule.
+    - Add comments throughout your code to explain your logic. Feel free to copy-paste the official USACO solution and improve it by adding more descriptive variable names and helpful comments.
     - Be _consistent_ about formatting your code, unlike the following snippet.
-      We don't care about which one you use (`a+b` or `a + b`), but pick one and
-      stick with it!
+      We don't care about which one you use (`a+b` or `a + b`), but pick one and stick with it throughout your entire code!
 
-      ```py
+     ```py
       a,b = (0,1)
       print(a+b)
       print(a + b)
       ```
+    - Use universally understandable variable names, especially for lower divisions (ie. Bronze and Silver). It is preferred to use longer / more descriptive variable names than what you would normally use in-contest since this is a learning resource.
+    - Remember to follow these guidelines when formatting your code for a pull request. These rules all have a specific purpose that makes the code readable to a general audience. Making sure that your pull request encompasses all these details would be desirable.
```

Comment:
wait what is even the point of this last bullet point? it should be very self explanatory that one should follow guidelines right? 😆 

other than that looks good!

Reply:
removing it rn

Reply:
done